ms-ledger/routes/types.go

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2022 Intel Corporation. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+package routes
+
+import (
+	"github.com/edgexfoundry/app-functions-sdk-go/v2/pkg/interfaces"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
+)
+
+type Controller struct {
+	lc                logger.LoggingClient
+	service           interfaces.ApplicationService
+	inventoryEndpoint string
+}
+
+func NewController(lc logger.LoggingClient, service interfaces.ApplicationService, inventoryEndpoint string) Controller {
+	return Controller{
+		lc:                lc,
+		service:           service,
+		inventoryEndpoint: inventoryEndpoint,
+	}
+}
+
+func (c *Controller) AddAllRoutes() int {
```

Comment:
Return error and modify `errorAddRouteHandler` to return error rather than logging. 
Logging and returning 1 was appropriate when it was part of main.